### DISCUSSION TITLE
how to achieve Galaxy brain achievement

### DISCUSSION THREAD

**Discussion by thunder-007** [Schweinepriester/github-profile-achievements#65] [Q&A] [answered]
I answered 4 discussions still I am not able to earn that achievement

**Comment by melvincwng** [comment, marked as answer]
@thunder-007, I have addressed the same issue in [#58](https://github.com/Schweinepriester/github-profile-achievements/discussions/58). 

Please head over to that thread and take a look.

Thanks!

**Reply by melvincwng** [reply]
@thunder-007, in short, you cannot mark your answers to your own questions or else there is a bug that would prevent your achievement badge from showing.

You need to unmark your answers of your own questions.

Then have another user mark one of your answers again, then the badge will automatically appear.

**Reply by thunder-007** [reply]
could you ask a question one I'll answer

**Reply by melvincwng** [reply]
@thunder-007, if this question has helped you, please 'Mark as answer' so the rest know this issue has been addressed.

Thank you.

**Reply by thunder-007** [reply]
done tq

**Comment by YouthguyZ** [comment]
Answer 2 discussions for Galaxy Brain
